Story/Friday13th/CoreFriday13th.cs

```diff
@@ -0,0 +1,825 @@
+        Story.KillQuest(6266, "Gonnagetcha", "Shrade Cultist");
+
+        // Stop the Ringleader 6267
+        Story.KillQuest(6267, "Gonnagetcha", "Bride of Shrade");
+        
+        // Defeat Shrade! 6268
+        Story.KillQuest(6268, "Gonnagetcha", "Shrade");
+
+        // Revitalize the Camp 6269
+        Story.KillQuest(6269, "Gonnagetcha", new[] { "Vengeful Ghost", "Shrade Cultist" });
+    }
+
+    public void Greymoor()
+    {
+        // Check progress and membership
+        if (Core.isCompletedBefore(6420) || !Core.IsMember && !CalculateFriday13())
+            return;
+
+        // Preload quests
+        Story.PreLoad(this);
+
+        // Tired of Gremlins 6409
+        Story.KillQuest(6409, "Greymoor", new[] { "Auto Gremlin", "Auto Gremlin", "Auto Gremlin" });
+
+        // I Don't Mean To Be CRUDE 6410
+        Story.KillQuest(6410, "Greymoor", "Oil Elemental");
+
+        // When I Say Jump... 6411
+        Story.KillQuest(6411, "Greymoor", "Huge Auto Gremlin");
+
+        // Energizing! 6412
+        Story.KillQuest(6412, "Greymoor", "Energy Elemental");
+
+        // Cabin in the Woods 6413
+        Story.MapItemQuest(6413, "Greymoor", 5912, 5);
+
+        // A Base-ic Solution 6414
+        Story.KillQuest(6414, "Greymoor", "Spooky Treeant");
+
+        // Surprise! 6415
+        Story.MapItemQuest(6415, "Greymoor", new[] { 5913, 5914 });
+
+        // First Aid Supplies 6416
+        Story.KillQuest(6416, "Greymoor", "Spooky Treeant");
+
+        // Wood for Trap? 6421
+        Story.KillQuest(6421, "Greymoor", "Spooky Treeant");
+
+        // Tool Time 6417
+        Story.MapItemQuest(6417, "Greymoor", 5915);
+
+        // Obstacle Course 6418
+        Story.MapItemQuest(6418, "Greymoor", 5916, 5);
+
+        // Barricades! 6419
+        Story.MapItemQuest(6419, "Greymoor", 5917, 4);
+
+        // Take Him Out! 6420
+        Story.KillQuest(6420, "Greymoor", "Shrade");
+    }
+
+    public void Puzzlebox()
+    {
+        // Check progress and membership
+        if (Core.isCompletedBefore(7399) || !Core.IsMember && !CalculateFriday13())
+            return;
+
+        // Preload quests
+        Story.PreLoad(this);
+
+        // Find the Souls 7394
+        Story.KillQuest(7394, "PuzzleBox", "Bones of the Doomed");
+
+        // The Blackened Heart 7395
+        Core.Join("PuzzleBox", "r5", "right", true);
+        Story.KillQuest(7395, "PuzzleBox", "Cursed Guardian");
+
+        // The Ancient Athame 7396
+        Core.Join("PuzzleBox", "r6", "right", true);
+        Story.KillQuest(7396, "PuzzleBox", "Cursed Guardian");
+
+        // The Withered Hand 7397
+        Core.Join("PuzzleBox", "r7", "right", true);
+        Story.KillQuest(7397, "PuzzleBox", "Cursed Guardian");
+
+        // The Bones 7398
+        Story.KillQuest(7398, "PuzzleBox", "Bones of the Doomed");
+
+        // The Puzzle 7399
+        Story.MapItemQuest(7399, "PuzzleBox", 7165);
+    }
+
+    public void Splatterwarshrade()
```

Comment:
+ there is a map called splatterwardage which is also prerequisite before this
this is all can be known from QuestData Sheet please check it regularly while making story scripts